### DISCUSSION TITLE
How to build with Bazel to generates xcodeproj files?

### DISCUSSION THREAD

**Discussion by abinhho** [google/mozc#1075] [Q&A] [answered]
Since GYP is under maintenance mode and not support AUX...
![Screenshot 2024-10-10 at 20 29 41](https://github.com/user-attachments/assets/222f223c-568c-417a-a5a8-f54e702f4dac)

**Bazel** is ok but how to config it to generates `.xcodeproj` files.

Ref: https://www.kodeco.com/31558158-building-with-bazel/lessons/18

Actually even `python3 build_mozc.py gyp` run successfully but `xcodeproj` files was not created. 

Please help!

**Comment by hiroyuki-komatsu** [comment, marked as answer]
Hi abinhho,

Mozc does not have rules to generate `.xcodeproj` files.

* You can build binaries without generating `.xcodeproj` files.
* You might want to use `rules_xcodeproj` if you need `.xcodeproj` files for your pupose.
 
 https://github.com/MobileNativeFoundation/rules_xcodeproj
